docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  app:
+    image: todo-app:dev
+    ports:
+      - 5000:5000
+    env_file: 
+      - .env
```

Comment:
You don't actually need this here since .env will be included in the bind mount.